### ISSUE TITLE
arc lable

### ISSUE THREAD

**Issue by alikharazm** [react-map/leaflet.migrationLayer#4]
hello,
thanks for your great library.
how can I add lable to arc(link)?
(in the middle of arc)

**Comment by duxiaodi** [comment]
Yes, of course,welcome any good ideas!
 



发自我的 iPhone

**Comment by hamzazar** [comment]
Hell, did you have a solution? For me, I think it will better if arc has a click or hover listener so I can add the information or action that I need. Is that doable?

**Comment by liuchuancong** [comment]
+1

**Comment by burrav93** [comment]
Yes is there a possibility to add tooltips for these arcs/lines and also the points? Thanks

**Comment by jharreman2012** [comment]
+1
